examples/aws-agave-validator/index.ts

```diff
@@ -3,9 +3,20 @@ import * as svmkit from "@pulumi/svmkit";
 import * as aws from "@pulumi/aws";
 import * as tls from "@pulumi/tls";
 
-const config = new pulumi.Config("svmkit");
-
-const ami = config.require("ami");
+const ami = pulumi.output(aws.ec2.getAmi({
```

Comment:
Would you make the AMI change a separate commit so it's obviously different from the version work?